changelogs/fragments/url-lookup-add-httpagent.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - url lookup - set default user agent to ``ansible-httpget``
```

Comment:
```suggestion
  - url lookup - set default user agent to ``ansible-httpget`` (https://github.com/ansible/ansible/pull/72324).
```

Reply:
You are correct. It's good to include the PR URL in the changelog.